solvebio/test/test_object.py

```diff
@@ -236,3 +236,21 @@ def test_object_add_tag_non_iterable_or_string(self, ObjectCreate, ObjectSave):
         tags = 1
         file_.tag(tags)
         self.assertTrue(file_.has_tag(tags))
+
+    def test_object_set_metadata(self):
+        folder = self.client.Object.get_or_create_by_full_path('~/test_folder_metadata',
+                                                               object_type='folder')
```

Comment:
since it is `get_or_create_` the metadata may already be set from a previous run of the test. can either create a new one each time...(use .create_by_full_path()) ...or one retrieving, clear the metadata dict and (do folder.save()) and then test that the set function works.

Reply:
Sure.